### PR TITLE
Siccar point/tutorial docs #1169

### PR DESCRIPTION
**Pull request by SiccarPoint**
Addresses #1169.

### CODE REVIEW

File: docs/source/user_guide/tutorials.rst [hunk 1]
```diff
@@ -15,8 +15,13 @@ Launch notebooks online
 
 Landlab Notebooks can be accessed online with the following link:
 `Binder <https://mybinder.org/v2/gh/landlab/landlab/release?filepath=notebooks/welcome.ipynb>`_.
-Here the notebooks are provided within a binder environment that includes
-Landlab.
+Here the notebooks are provided within a binder online environment that
+includes Landlab.
+
+The welcome page on Binder provides onward links to most of our tutorials.
```
Comment (gregtucker) [inline docs/source/user_guide/tutorials.rst]: This is a helpful addition, thanks

File: notebooks/tutorials/syllabus.ipynb [hunk 2]
```diff
@@ -17,6 +17,8 @@
     "\n",
     "**We highly recommend reading it before starting on the steps below.**\n",
     "\n",
+    "Before starting the syllabus, consider reading our very basic [introduction to Python](python_intro/python_intro.ipynb) if you think of yourself as a Python novice.",
```
Comment (gregtucker) [inline notebooks/tutorials/syllabus.ipynb]: good addition

File: notebooks/welcome.ipynb [hunk 3]
```diff
@@ -22,16 +22,16 @@
     "## Other useful links\n",
     "- [The Landlab Documentation](https://landlab.readthedocs.io/en/latest/)\n",
     "- [The Landlab code base](https://github.com/landlab/landlab)\n",
+    "- [The Landlab user guide](https://landlab.readthedocs.io/en/latest/user_guide/)\n"
```
Comment (gregtucker) [inline notebooks/welcome.ipynb]: yes!

Comment (kbarnhart) [reply notebooks/welcome.ipynb]: agreed. though a missing comma at the end of this line broke the tests. I've fixed it.

**Comment by kbarnhart** [comment]
Only errors are transient link check errors with the docs. Merging.

**Pull request merged**